Source/Bogus/Distributions/Gaussian/ExtensionsForRandomizer.cs

```diff
@@ -110,9 +107,6 @@ public static int GaussianInt(this Randomizer rnd, double mean, double standardD
       /// </code>
       /// </example>
       /// </summary>
-      /// <param name="rnd"></param>
-      /// <param name="mean"></param>
-      /// <param name="standardDeviation"></param>
```

Comment:
Might be best to copy/paste the parameter XML docs from the other similar methods instead of removing.

Reply:
done